pkg/daemon/daemon.go

```diff
@@ -248,7 +248,7 @@ func (d *daemon) AddPeriodicUpdate() {
 				pod.Annotations[v1.NetworkAttachmentAnnot] = string(netAnnotations)
 			}
 
-			guidList = append(guidList, guidAddr)
+			guidList = append(guidList, net.HardwareAddr(guidAddr))
```

Comment:
why not GUID?

Reply:
@moshe010  Do you prefer to change all the net.HardwareAddress to GUID, including the sm plugin parameters?

Reply:
I see just add a comment why you add it  as net.HardwareAddr. and we change it in another PR